sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added Xmethods that help GDB to support reading local_accessors on GPU from SLM. [ddfc5220afaa]
+- Enabled passing local accessors to ESIMD kernel and via `invoke_simd()` API, enabled usage of `get_pointer()` and `operator[]` for accessors in ESIMD kernel. The support of accessor and local_accessor is still limited comparing to SYCL. [82ae85d12bf0]
+- Implement unpadding for 2d block load/store ESIMD API. [9be8a306d0eb]
+- Enabled improved code location for SYCL extensions. [4d73490220c8]
+- Added SYCL 2020 image classes. [9035cdb0898d][1a3e99307e4f]
+- Added interfaces for SYCL 2020 image accessors, only the host accessors interfaces are actually functional. [a58d8e3e88c0]
+- Added XPTI notifications for SYCL 2020 images as well as for the associated accessor classes. [8c535276785d][73d5c04d0305]
+- Added new FPGA loop attribute `enable_loop_pipelining`. [4147b8ded75f]
+- Started to print device aspects in `sycl-ls --verbose`. [e32ab1759fed]
+- Added tangle/opportunistic algorithms. [29e629e00e83]
+- Implemented `info::device::backend_version` query. [df743a5c97e1]
+- Added `fixed_size_group` support to algorithms. [01ac0338bfa9]
+- Added simple `abs(int)` to imf libdevice. [2a3aaeea5be8]
+- Added `ballot_group` support to algorithms. [70a35de4eacb]
+- Implemented interoperability support for images for the Level Zero backend. [836ceec36b57]
+- Added marray support for math built-in functions. [a7a6de2e40a2][c274f40588ff][2953d2548791]
+- Enabled inline assembly support in ESIMD kernels. [f5a062dd057f][48abe38c0bee]
+- Enhanced interop queue interface to choose standard or immediate commandlists. [0ff3ec40e241]
+- Enabled double type for `atomic_update()` ESIMD API. [98c443543c72]
+- Added support for `addc` and `subb` operations for ESIMD. [0a098933d9ba]
+
+### Documentation
+- Added experimental SYCL bindless images extension [`sycl_ext_oneapi_bindless_images`](doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) proposal.
+- Added [`sycl_ext_oneapi_work_group_local`](doc/extensions/proposed/sycl_ext_oneapi_work_group_local.asciidoc) extension proposal. [1d6cff1a8e5a]
+- Moved [sycl_ext_oneapi_device_global](doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc) from proposed to experimental. [d812d1e13ab5]
+- Added [`DeviceConfigFile`](doc/design/DeviceConfigFile.md) design document. [3f4b778dd163]
+- Added [`sycl_ext_oneapi_address_cast`](doc/extensions/proposed/sycl_ext_oneapi_address_cast.asciidoc) extension. [9cab5598bc82]
+- Added [`sycl_ext_intel_grf_size`](doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc) extension. [83d099758770]
+- Added proposal for [`sycl_ext_oneapi_append_and_shift`](doc/extensions/proposed/sycl_ext_oneapi_append_and_shift.asciidoc) extension. [1a283acaac3c]
+- Updated description of accessor-based memory APIs in [`sycl_ext_intel_esimd`](sycl/doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md). [9e5889918277]
```

Comment:
That link leads to File Not Found error page. Not sure what is the right solution here: relative links will be outdated in the future (well, some of them are outdated even now), and putting a link to a certain commit hash means that users won't see the latest version of the extension.

Perhaps the latter is not so bad? Because user was following release notes for a certain release and therefore extension versions should also be fixed

Reply:
Link to a certain commit hash sounds good, fixed all links to point to exact hash and checked links.